core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -331,7 +334,7 @@ private LocalQueryRunner(
 
         this.metadata = new MetadataManager(
                 featuresConfig,
-                new SessionPropertyManager(new SystemSessionProperties(new QueryManagerConfig(), taskManagerConfig, new MemoryManagerConfig(), featuresConfig, new NodeMemoryConfig(), new DynamicFilterConfig(), new NodeSchedulerConfig())),
+                new SessionPropertyManager(requireNonNull(systemSessionProperties, "systemSessionProperties is null")),
```

Comment:
Concatenate `systemSessionProperties` with:
```
new SystemSessionProperties(
  new QueryManagerConfig(),
  taskManagerConfig,
  new MemoryManagerConfig(),
  featuresConfig, new NodeMemoryConfig(),
  new DynamicFilterConfig(),
  new NodeSchedulerConfig())
```
`SystemSessionProperties` should always be present

Reply:
`getSystemSessionProperties` can be removed in that case,